### DISCUSSION TITLE
Run addon only in the editor

### DISCUSSION THREAD

**Discussion by scratchusernamemrtbts** [ScratchAddons/ScratchAddons#1739] [Help] [answered]
Is there a way to edit the addon.JSON file for the addon to only run when in the editor
Like setting matches to https://scratch.mit.edu/projects/*/editor
Will this work

**Comment by scratchusernamemrtbts** [comment]
This will help on #1738 
So when viewing a project it doesn’t run the addon
Let it run only when I open the editor

**Comment by mxmou** [comment, marked as answer]
No. The editor is technically the same page as the project page, if you want to do something only in the editor you need to implement that using JavaScript.

**Reply by scratchusernamemrtbts** [reply]
Thank you
